frontend/src/App.scss

```diff
@@ -66,23 +66,28 @@ div.bootstrap-table {
   word-break: break-word;
 }
 
+select.form-control.is-invalid {
+  background-image: none;
+  padding-right: 12px;
+}
+
 //by default highlight required fields in blue.
-.form-group.required input {
+.form-group.required input:not(.is-invalid) {
```

Comment:
this blue highlight should never override the red highlight of an input in error.